pytest_html/plugin.py

```diff
@@ -71,7 +71,9 @@ def pytest_unconfigure(config):
 
 def data_uri(content, mime_type='text/plain', charset='utf-8'):
     if PY3:
-        data = b64encode(content.encode(charset)).decode('ascii')
+        if not isinstance(content, bytes):
+            content = content.encode(charset)
```

Comment:
This will work, but is generally frowned upon - I'm not familiar enough with the context though to say if it'd make more sense to do it like this, or e.g. add a `data_uri_bytes`.